### ISSUE TITLE
consider changing .INI files to .TOML

### ISSUE THREAD

**Issue by nickzoic** [CountESS-Project/CountESS#28]
[TOML](https://toml.io/en/) does pretty much the same as the extra string-escaping stuff we sprinkle onto the .INI files, and probably fixes a few things we didn't think of 
while still having the positive things we were after.
See also #15

**Comment by afrubin** [comment]
I agree that TOML is a better format to target than INI as it's a bit more "modern" and is likely to give us fewer headaches later on.

If we want to make this change, I think now is the time before we have wider adoption. I'm assuming that conversion from existing CountESS .INI files to TOML is reasonably straightforward and can be accomplished with a simple standalone script, or we can retain the ability to read .INI files and just start saving as TOML.

**Comment by nickzoic** [comment]
yeah I reckon I can just keep the .INI reader and make it read & write TOML by default.
